samples/apps/copilot-chat-app/webapi/CopilotChat/Models/DocumentMessageContent.cs

```diff
@@ -10,17 +12,49 @@ namespace SemanticKernel.Service.CopilotChat.Models;
 /// </summary>
 public class DocumentMessageContent
 {
+    public class Document
```

Comment:
it seems our convention is giving public classes their own file.

Reply:
Making it private

Reply:
Separating it to its own file